Vegacapsule system-test pipe does not stop the network gracefully

The pipeline currently goes `start-network -> run-tests -> suck-up-artefacts`

What I am seeing is that when I pull the build-artefacts onto my machine from the CI the tendermint block/state databases are in a bad way and I cannot replay the chain locally:
`I[2022-05-16|13:51:22.975] ABCI Replay Blocks                           module=consensus appHeight=7047 storeHeight=7131 stateHeight=7135
panic: StateBlockHeight (7135) > StoreBlockHeight (7131)`

I experimented by changing `make test` to both run the tests and call `vegacapsule network stop` and it seemed to fix the issue. I imagine us trying to copy files while the chain is still runnning is probably not a good idea, and that pipeline needs a new stage which stops the network once the tests have finished.

--- a/vars/capsuleSystemTests.groovy
+++ b/vars/capsuleSystemTests.groovy
@@ -177,6 +177,12 @@ void call(Map additionalConfig) {
       }
     }
   } finally {
+    stage('Stop network') {
+      dir('tests') {
+        sh './vegacapsule network stop --home-path ' + testDirectoryPath + '/testnet'
+      }
+    }
+
     stage('Archive network logs') {
       dir('tests') {
         if (config.printNetworkLogs) {

